src/main/java/maybeweijun/parser/Parser.java

```diff
@@ -0,0 +1,187 @@
+package maybeweijun.parser;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+import maybeweijun.exception.maybeweijunException;
+import maybeweijun.task.Deadline;
+import maybeweijun.task.Event;
+import maybeweijun.task.Task;
+import maybeweijun.task.TaskList;
+import maybeweijun.task.Todo;
+import maybeweijun.ui.Ui;
+
+public class Parser {
+
+    public static boolean process(String input, TaskList tasks, Ui ui) throws maybeweijunException {
```

Comment:
Maybe you could consider changing the method name to sound like booleans as it is a boolean method?
```suggestion
    public static boolean hasProcessed(String input, TaskList tasks, Ui ui) throws maybeweijunException {
```